src/boot/ksdk1.1.0/devAS7262.h

```diff
@@ -44,3 +46,6 @@ WarpStatus	readSensorSignalAS7262(WarpTypeMask signal,
 					WarpSignalAccuracy accuracy,
 					WarpSignalReliability reliability,
 					WarpSignalNoise noise);
+
+WarpStatus LEDonAS7262(void);
```

Comment:
Please see the coding conventions at https://github.com/physical-computation/Warp-firmware/blob/master/doc/README-CodingConventions.md